src/rlgl.h

```diff
@@ -815,7 +815,19 @@ RLAPI void rlLoadDrawQuad(void);     // Load and draw a quad
     #include "external/glad.h"          // GLAD extensions loading library, includes OpenGL headers
 #endif
 
-#if defined(GRAPHICS_API_OPENGL_ES3)
+#if defined(PLATFORM_IOS)
```

Comment:
If possible I'd prefer to keep `rlgl` as platform-agnostic as possible, would it be possible to move this configuration to the build system or `rcore.c` instead of manage it here?